src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CFTTaskMapperTest.java

```diff
@@ -2449,6 +2449,14 @@ void reconfigure_config_additional_attributes_dmn_fields() {
 
     }
 
+    @Test
+    void do_not_reconfigure_additional_attributes_fields_if_empty_list_returned() {
+        TaskResource taskResource = createTaskResource();
+        List<ConfigurationDmnEvaluationResponse> additionalPropertyDMNResponse = new ArrayList<>();
+        cftTaskMapper.reconfigureAdditionalTaskAttribute(taskResource, additionalPropertyDMNResponse);
+        assertNull(taskResource.getAdditionalProperties());
+    }
```

Comment:
This is ok for now to address the problem in prod.  However, we should review the functional tests to ensure we are not changing the json response when there are empty additional properties.  It's not a breaking change in the API as it's an agreed field but could break some clients if coded badly.